doc/Index.md

```diff
@@ -2,6 +2,7 @@ VS Project System Documentation
 ===============================
 
 - [Introduction](overview/intro.md)
+- [Change log](changelog.md)
```

Comment:
The top few links in this file are very important to read. Can we move this down the list below the core "onboarding" documents?